spec/unit/generator/xcconfig/aggregate_xcconfig_spec.rb

```diff
@@ -107,8 +111,20 @@ def spec
             @xcconfig.to_hash['OTHER_CFLAGS'].should == expected
           end
 
-          it 'links the pod targets with the aggregate target' do
-            @xcconfig.to_hash['OTHER_LDFLAGS'].should.include '-l"Pods-BananaLib"'
+          describe 'with a scoped pod target' do
+            def pod_target(spec)
```

Comment:
But it is, it is defined in the root context and used in the before block there, but overwritten on this sub-context.